app/views/foods/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Recipe App</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", async: true %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body>
+
+    <%= render "partials/nav" %>
+    
+    <section class="body">
+      <div class="add-btn">
+        <%= link_to 'foods/new', :remote => true do %>
+          <input type="button" class="btn btn-primary" value="Add Food">
+        <% end %>
+      </div>
+
+      <div class="table">
+        <table class="table table-bordered">
+        <tr>
+          <th>Food</th>
+          <th>Measurement Unit</th>
+          <th>Unit Price</th>
+          <th>Action</th>
+        </tr>
+
+        <% @foods.each do |food| %>
+        <% if can? :manage, food %>
+          <tr>
+            <td><%=food.name%></td>
+            <td><%=food.measurement_unit%></td>
+            <td><%=food.price%>$</td>
+            <td>
+                <%= button_to "Delete", food_path(food.id), html: {'data-turbo' => "false"}, :method => :delete, class: "btn btn-warning" %>
+            </td>
```

Comment:
- [ ] Nice one so far, users can see foods and public recipes created by other users but can not delete them. However, the delete button for the food still shows up even though it does not delete as long as you are not the one that created it. It would be nice to use `if` statement to display the delete button only if you are the one that created the food. See the image below:

Julie created the food | May is user 2
:---:|:---:
![](https://i.imgur.com/puia2n6.png) | ![](https://i.imgur.com/pEU2Hdt.png)

Reply:
Implemented